docs/.vuepress/sidebar/en.ts

```diff
@@ -114,6 +114,34 @@ export const enSidebar = sidebar({
         "/api/js/front-end",
       ],
     },
+    {
+      text: "JavaScript Ethers V6 SDK",
```

Comment:
I find it would make it easier to follow if both the v5 and v6 docs are under the same "JS SDK" category on the sidebar, then we have a "JS SDK ev5" subcategory under it with all the docs from before and another "JS SDK ev6" category with these new docs, as:

JS SDK 
--ethersv5 
----old docs as before
--ethersv6
----these new docs

Reply:
Suggested changes are applied.